common/src/main/java/org/apache/falcon/entity/v0/EntityGraph.java

```diff
@@ -200,6 +200,15 @@ public void onReload(Entity entity) throws FalconException {
                 feedEdges.add(dbNode);
                 dbEdges.add(feedNode);
             }
+            if (FeedHelper.isExportEnabled(cluster)) {
```

Comment:
https://github.com/apache/falcon/pull/165/files seems to have same changes for EntityGraph.java, can you make sure only one of the two pull requests have this change?